src/test/kotlin/heist/make/sdk/kt/sse/EventSourceTest.kt

```diff
@@ -22,7 +22,7 @@ import kotlin.test.Test
                     "data: Yahya!\n" +
                     "data: this is data\n" +
                     "data: I am a message\n" +
-                    ", type= message)\n", e.toString()
+                    ", type= message)\n", Json.encodeToString(e)
```

Comment:
This doesn't work?

Reply:
I would just serialize the result and check against a know response.

Reply:
It gives me an error saying that I can't use serialization in the EventSourceTest class
This is the error that I get now. 

kotlinx.serialization.SerializationException: Class 'class heist.make.sdk.kt.sse.EventSourceTest$testEventSource$1 (Kotlin reflection is not available)' is not registered for polymorphic serialization in the scope of 'Function1'.
Mark the base class as 'sealed' or register the serializer explicitly.